composer.json

```diff
@@ -21,36 +21,37 @@
         }
     ],
     "require": {
-        "php": "^5.5.9 || ^7.0",
-        "paragonie/random_compat": "^1 || ^2 || ^9",
-        "symfony/form": "^2.8 || ^3.0 || ^4.0",
-        "symfony/framework-bundle": "^2.8 || ^3.0 || ^4.0",
-        "symfony/security-bundle": "^2.8 || ^3.0 || ^4.0",
-        "symfony/templating": "^2.8 || ^3.0 || ^4.0",
-        "symfony/translation": "^2.8 || ^3.0 || ^4.0",
-        "symfony/twig-bundle": "^2.8 || ^3.0 || ^4.0",
-        "symfony/validator": "^2.8 || ^3.0 || ^4.0",
-        "twig/twig": "^1.28 || ^2.0"
+        "php": "^7.2",
+        "doctrine/persistence": "^1.3",
+        "symfony/form": "^3.4 || ^4.4",
+        "symfony/framework-bundle": "^3.4 || ^4.4",
+        "symfony/security-bundle": "^3.4 || ^4.4",
+        "symfony/templating": "^3.4 || ^4.4",
+        "symfony/translation": "^3.4 || ^4.4",
+        "symfony/twig-bundle": "^3.4 || ^4.4",
+        "symfony/validator": "^3.4 || ^4.4",
+        "twig/twig": "^2.0 || ^3.0"
     },
     "conflict": {
-        "doctrine/doctrine-bundle": "<1.3",
         "symfony/doctrine-bridge": "<2.7"
     },
     "require-dev": {
-        "doctrine/doctrine-bundle": "^1.3",
-        "friendsofphp/php-cs-fixer": "^2.2",
-        "phpunit/phpunit": "^4.8.35 || ^5.7.11 || ^6.5",
-        "swiftmailer/swiftmailer": "^4.3 || ^5.0 || ^6.0",
-        "symfony/console": "^2.8 || ^3.0 || ^4.0",
-        "symfony/phpunit-bridge": "^2.8 || ^3.0 || ^4.0",
-        "symfony/yaml": "^2.8 || ^3.0 || ^4.0"
+        "doctrine/doctrine-bundle": "^1.12 || ^2.0",
+        "friendsofphp/php-cs-fixer": "^2.15",
+        "phpunit/phpunit": "^7.0 ||  ^8.0",
+        "swiftmailer/swiftmailer": "^6.0",
+        "symfony/cache": "^3.4 || ^4.4",
+        "symfony/config": "^3.4 || ^4.4",
```

Comment:
`symfony/config` should be a dependency. We use it in our DI layer for the Configuration class.